### PR TITLE
Feature/hn 5 create the model classes needed

### PR DESCRIPTION
**Pull request by alexsandermeneses**

### CODE REVIEW

File: src/main/java/com/happeningnow/Model/Artist.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.happeningnow.Model;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.UuidGenerator;
+import java.util.List;
+import java.util.UUID;
+
+@Data
```
Comment (arnaldotecadm) [inline src/main/java/com/happeningnow/Model/Artist.java:9]: if you are using @Data, then you do not need @AllArgsConstructor and @NoArgsConstructor, I guess

File: src/main/java/com/happeningnow/Model/Artist.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+@AllArgsConstructor
+@NoArgsConstructor
+
+@Entity
+@Table(name = "artist")
+public class Artist {
+    @Id
+    @UuidGenerator
+    private UUID id; // one to many
```
Comment (arnaldotecadm) [inline src/main/java/com/happeningnow/Model/Artist.java:18]: no need to comment the properties, remove them.

File: target/classes/application.yaml
Comment (arnaldotecadm) [inline target/classes/application.yaml:1]: Add the folder target to the .gitinore file

**Review by arnaldotecadm — CHANGES_REQUESTED** [review]

File: target/classes/application.yaml
Comment (alexsandermeneses) [reply target/classes/application.yaml:1]: ok

File: src/main/java/com/happeningnow/Model/Artist.java (hunk 1 above)
Comment (alexsandermeneses) [reply src/main/java/com/happeningnow/Model/Artist.java:9]: ok

File: src/main/java/com/happeningnow/Model/Artist.java (hunk 2 above)
Comment (alexsandermeneses) [reply src/main/java/com/happeningnow/Model/Artist.java:18]: ok

File: src/main/java/com/happeningnow/model/Event.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+
+@Entity
+@Table(name = "event")
+public class Event {
```
Comment (arnaldotecadm) [inline src/main/java/com/happeningnow/model/Event.java:15]: Looking to the proposed model, itis missing a reference for the organizer in this entity

Comment (alexsandermeneses) [reply src/main/java/com/happeningnow/model/Event.java:15]: ok

**Review by arnaldotecadm — COMMENTED** [review]
LGTM

**Review by arnaldotecadm — APPROVED** [review]

**Pull request merged**